packages/ember-glimmer/lib/components/curly-component.js

```diff
@@ -1,6 +1,7 @@
 import { StatementSyntax, ValueReference } from 'glimmer-runtime';
 import { AttributeBindingReference, RootReference, applyClassNameBinding } from '../utils/references';
 import EmptyObject from 'ember-metal/empty_object';
+import Dict from 'ember-metal/dictionary';
```

Comment:
We do want the object to have a fixed shape though even when some of those meta info are not present.